src/tests.zig

```diff
@@ -19,7 +21,7 @@ pub fn generateKeypair() !void {
     }
 }
 
-pub fn signAndVerifyEcdsa() !void {
+test "signAndVerifyEcdsa" {
```

Comment:
it's was more example, not a test (test for ecdsa already wrotten in a secp256k1), but anyway it can be as a test-example

Reply:
I can rename the file as examples.zig to make it more explicit. Wdyt?

Reply:
i think its okay left like that, global test file with test examples